Automattic/sensei
# Simplify backend layers.

Fixes #5850

### Changes proposed in this Pull Request
* Removed most of the domain classes (except from the interface and implementations for tasks).
* Removed mapper and moved array structure into each individual provider.
* Fixed tests and integration with controller.

### Testing instructions
- Check code changes (suggested to follow the separate commits).
- Check that `curl -s -X GET --user aaron:development 'http://test2.local/wp-json/sensei-internal/v1/home' | jq .` returns expected structure.
